repairnator/docker-images/kubernetes/README.md

```diff
@@ -0,0 +1,141 @@
+# Deploy a high avaliable Repairnator in Kubernetes
+
+*On going pull request*
+
+In order to make Repairnator highly available, it might be good to have configurations and tools to deploy Repairnator in Kubernetes, a preliminary idea here:
+
+- A queue system like ActiveMQ to save build ids
+- A K8s cron job running repairnator scaner to get latest build ids and add them into the queue
```

Comment:
Why use the scanner instead of the realtime scanner? That would make the cron job unnecessary.

Reply:
That's one of the old information part . There are no cron job used in here :) so that part of the readme need to be removed . Then again the requirement for my task was minimal change or no change to the source code and after some checking on the RT scanner it seems that it uses docker images to validate/check the builds it found which is unfavorable since it would mean that we will be running docker inside docker. So was our thoughts about this after My own discussion with @Gluzhang

Reply:
> Then again the requirement for my task was minimal change or no change to the source code and after some checking on the RT scanner it seems that it uses docker images to validate/check the builds it found which is unfavorable since it would mean that we will be running docker inside docker.

Wouldn't this be mitigated by running the queue system and starting up pods? Or am I misunderstanding how K8S work?

Reply:
No, it seems we won't be able to escape docker container running inside the RTscanner pod container if we deploy it. The queue is only for putting jobs but the app itself will do the same thing inside the pod, which is running images. :)

Reply:
Ahh, okay. Then I get it. Thanks for the response.

Reply:
No problem man :).